### ISSUE TITLE
Improve the csv reading error display

### ISSUE THREAD

**Issue by ElMigu17** [ElMigu17/TAP#3]
When reading a csv file and making the data structures from it's infos, it might occur some specific errors. To be possible to fix them, the error are collected and displayed when the "Erros de Leitura" button is clicked (witch appears when the csv toggle is activated). Currently, this display is kind of ugly. Can you improve it?

**Comment by luisteixeira13** [comment]
Hello good morning, I would like to work on this issue, can I?

**Comment by ElMigu17** [comment]
Yes, of course. If you have any doubt or difficulty, send me a message or e-mail.
